docs/building-spark.md

```diff
@@ -153,7 +153,8 @@ Thus, the full flow for running continuous-compilation of the `core` submodule m
 
 # Using With IntelliJ IDEA
```

Comment:
@pwendell Done. The other half of this change is then updating the wiki with some additional information. You can see my proposed text above and the tidbits I think should be on the wiki. Feel free to integrate whatever of it you see fit, however you see fit.